.travis.yml

```diff
@@ -1,18 +1,17 @@
 language: python
 sudo: false
-# We need ghostscript at least 9.15 and japanese minch fonts
-addons:
-    apt:
-        sources:
-        - debian-sid
-        packages:
-        - ghostscript
-        - fonts-takao-mincho
 env:
     - TOXENV=py27
+    - TOXENV=py33
+    - TOXENV=py34
 install:
     - travis_retry pip install tox
 script:
     - tox
 notifications:
     email: false
+before_install:
+    - wget http://downloads.ghostscript.com/public/binaries/ghostscript-9.15-linux-x86_64.tgz
+    - tar -xvf ghostscript-9.15-linux-x86_64.tgz
+    - mv ghostscript-9.15-linux-x86_64/gs-915-linux_x86_64 ghostscript-9.15-linux-x86_64/gs
+    - export PATH=$PWD/ghostscript-9.15-linux-x86_64:$PATH
```

Comment:
Why not using packaged version of ghostscript? Also, we need japanese fonts to properly test japanese pdf rendering.